Lexis+.js

```diff
@@ -0,0 +1,267 @@
+			//  This could break titles in future. I do remove P.L. below though.
+			if (title === title.toUpperCase()) title = ZU.capitalizeTitle(title.toLowerCase(), true); // Some acts are capitalized
+
+			// Remove some unnecessary information
+			var cleanedTitle = title;
+			var pLCite = title.match(/(\d+ p\.l\. \d+)/i);
+			var statCite = title.match(/(\d+ stat\. \d+)/i);
+			var enactedCite = title.match(/\d+ enacted [a-zA-Z0-9.]+ \d+/gi);
+			var part = title.match(/(part \d+(?: of \d+)?)/i);
+			if (pLCite) cleanedTitle = cleanedTitle.replace(pLCite[1], '');
+			if (statCite) cleanedTitle = cleanedTitle.replace(statCite[1], '');
+			if (part) cleanedTitle = cleanedTitle.replace(part[1], '');
+			if (enactedCite) {
+				// Remove every enacted cite
+				for (var value of Object.values(enactedCite)) {
+					cleanedTitle = cleanedTitle.replace(value, '');
+				}
+			}
+			cleanedTitle = cleanedTitle.replace(/(^\s*,)|(,\s*$)/g, ''); // Trim commas and whitespace
+			cleanedTitle = cleanedTitle.replace(/(^\s*,)|(,\s*$)/g, ''); // Another one
+			Zotero.debug(cleanedTitle);
+			if (ZU.trim(cleanedTitle) === "") { // If the title's empty now, put it as the highest precedence citation in the title
+				Zotero.debug("ha");
+				if (pLCite) cleanedTitle = pLCite[1];
+				else if (statCite) cleanedTitle = statCite[1];
+				else if (enactedCite) {
+					cleanedTitle = enactedCite[0] + " & " + (Object.keys(enactedCite).length - 1) + " more";
+				}
+			}
+			newStatute.title = cleanedTitle;
+
+			// Reporter & citation formatting
+			var statutesAtLarge, publicLawNo;
+			var potentialReporter = text(doc, 'a.SS_ActiveRptr');
+			if (potentialReporter) { // Sometimes Lexis is weird and doesn't give an ActiveRptr
+				if (potentialReporter.match(/stat\./i)) statutesAtLarge = potentialReporter;
+				else if (potentialReporter.match(/pub\./i)
+				|| potentialReporter.match(/p\.l\./i)) {
+					publicLawNo = potentialReporter;
+				}
+			}
+
+			var otherReporters = doc.querySelectorAll('span.SS_NonPaginatedRptr');
+
+			for (var i = 0; i < otherReporters.length; i++) {
+				var nextReporter = otherReporters[i].textContent;
+				if (nextReporter.match(/stat\./i)) statutesAtLarge = nextReporter;
+				else if (nextReporter.match(/pub\./i)
+				|| nextReporter.match(/p\.l\./i)) {
+					publicLawNo = nextReporter;
+				}
+			}
+
+			// Turn publicLawNo into the public law fields
+			if (publicLawNo.match(/\d+-\d+/)) { // Ex. P.L. 115-164
+				var numPos = publicLawNo.search(/\d+-\d+/);
+				newStatute.publicLawNumber = publicLawNo.substring(numPos, publicLawNo.substring(numPos + 1).indexOf(' ')); // Gets 115-164
+
+				newStatute.session = newStatute.publicLawNumber.substring(0, newStatute.publicLawNumber.indexOf('-'));
+			}
+			else { // Ex. 115 P.L. 164 or 115 Pub. L. No. 164
+				var pLNumbers = publicLawNo.match(/(\d+) p\.l\. (\d+)/i);
+				newStatute.session = pLNumbers[1];
```

Comment:
you're sure `pLNumbers` always exists here? Otherwise import will error out at this point.

Reply:
Yes, reporters are guaranteed.